pkg/scheduler/testing/wrappers.go

```diff
@@ -1014,6 +1015,9 @@ func (wrapper *ResourceClaimWrapper) Request(deviceClassName string) *ResourceCl
 
 // Allocation sets the allocation of the inner object.
 func (wrapper *ResourceClaimWrapper) Allocation(allocation *resourceapi.AllocationResult) *ResourceClaimWrapper {
+	if !slices.Contains(wrapper.ResourceClaim.Finalizers, resourceapi.Finalizer) {
+		wrapper.ResourceClaim.Finalizers = append(wrapper.ResourceClaim.Finalizers, resourceapi.Finalizer, resourceapi.Finalizer)
```

Comment:
Why do we append it twice?

Reply:
Ooops. Must be a copy-and-paste error - will fix.

Reply:
Once this one is fixed, I am good with this PR

Reply:
Fixed.

/assign @thockin